sklearn/utils/extmath.py

```diff
@@ -28,15 +29,15 @@
 from ..exceptions import NonBLASDotWarning
 
 
-def norm(x):
+@deprecated("sklearn.utils.extmath.norm was deprecated in version 0.19"
+            "and will be removed in 0.21. Use scipy.linalg.norm instead.")
+def norm(x, *args, **kwargs):
```

Comment:
The function signature for `norm()` changed over scipy versions, so I'm just passing it to the original function. WDYT?